Week4/homework/ex1-aggregates/query-execution.js

```diff
@@ -0,0 +1,95 @@
+const seedDatabase = require("../ex1-aggregates/seedDatabase");
+const MongoClient = require("mongodb").MongoClient;
+const env = require("dotenv").config();
+
+// This runs everything
+async function main() {
+  const client = new MongoClient(process.env.MONGODB_URL, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  });
+
+  const db = client.db("databaseWeek4").collection("population_pyramid");
+
+  try {
+    await client.connect();
+
+    await totalPopulationPerCountry(db, "Netherlands");
+
+    await totalPopulationPerContinent(db, "2020", "100+");
+
+    await seedDatabase(client);
+  } catch (err) {
+    console.error(err);
+  } finally {
+    client.close();
+  }
+}
+
+// Exercise 1.2 per country
+async function totalPopulationPerCountry(db, country) {
+  const pipeline = [
+    {
+      $match: {
+        Country: `${country}`,
+      },
+    },
+    {
+      $addFields: {
+        totalPopulation: {
+          $add: [{ $toInt: "$M" }, { $toInt: "$F" }],
```

Comment:
```suggestion
          $add: [{ "$M" }, {  "$F" }],
```

It's better to just pass an integer to the function